specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/createCspSubscription.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "billingAccountName": "cd3ae1d5-9cb9-5950-9ffc-bc522547dbad:34756dd9-25b9-40c6-bc85-c25f68bff94f_2018-09-30",
```

Comment:
Is this a valid billing account name? Our examples should be semi-realistic. In the other examples the billingAccountName is just a Guid.

Reply:
Both the below formats are supported  for billingAccountName:
* cd3ae1d5-9cb9-5950-9ffc-bc522547dbad or
* cd3ae1d5-9cb9-5950-9ffc-bc522547dbad:34756dd9-25b9-40c6-bc85-c25f68bff94f_2018-09-30 (accountId:organizationId).